app/controllers/v2/editions_controller.rb

```diff
@@ -0,0 +1,64 @@
+module V2
+  class EditionsController < ApplicationController
+    def index
+      query = Queries::KeysetPagination.new(
+        Queries::GetEditions.new(
+          fields: query_params[:fields],
+          filters: filters,
+        ),
+        **pagination_params
+      )
```

Comment:
Good point, I've pushed some commits for this now.

Also I've implemented the feature to only show other pagination links when they are results available!